app/app.rb

```diff
@@ -0,0 +1,78 @@
+ENV['RACK_ENV'] ||= 'development'
+require_relative 'data_mapper_setup'
+require 'sinatra/base'
+require 'sinatra/flash'
+require 'bcrypt'
+
+class Chitter < Sinatra::Base
+  use Rack::MethodOverride
+  enable :sessions
+  set :session_secret, 'super_secret'
+
+  register Sinatra::Flash
+
+  attr_reader :name
+
+  helpers do
+    def current_user
+      @current_user ||=User.get(session[:user_id])
+    end
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  get '/sign_up' do
+    erb :sign_up
+  end
+
+  get '/chitters' do
+    erb :chitters
+  end
+
+  get '/peeps' do
+    @peeps = Peep.all(order: :created_at.desc)
```

Comment:
Could you perhaps pull this particular way of querying Peeps back into the model, so this line could have a nicer interface, something like: `@peeps = Peep.by_date`